### DISCUSSION TITLE
Suggestion: Minimize muted notes notification

### DISCUSSION THREAD

**Discussion by holokat** [damus-io/damus#1220] [General]
Right now the muted notes notification stands out quite a bit. I think it would be nice to minimize the notice and remove the bright link so it's easier to ignore those npubs entirely. Twitter is a good example IMO on how to handle muted accounts replies. 

Current on iphone se 2020:
![IMG_2293F3B8EAAD-1](https://github.com/damus-io/damus/assets/131526701/95ee3397-1732-4a51-a940-da641fb300ed)

Here is Twitter (white link, muted text). I think it could be even less emphasizing by removing outline and just making it a text line. 
![CleanShot 0005-06-01 at 15 07 43@2x](https://github.com/damus-io/damus/assets/131526701/e07fdc36-ade3-45df-8aee-123f8831a833)

**Comment by jb55** [comment]
On Wed, May 31, 2023 at 11:09:28PM -0700, Karnage wrote:
>Right now the muted notes notification stands out quite a bit. I think it would be nice to minimize the notice and remove the bright link so it's easier to ignore those npubs entirely. Twitter is a good example IMO on how to handle muted accounts replies.

agreed
